src/chi/javascript/components/sidenav.js

```diff
@@ -487,7 +491,7 @@ class Sidenav extends Component {
       this._addEventHandler(
```

Comment:
This event handler is added as many times as the drawer is shown. Chi components automatically remove event handlers only when the component is disposed. It would be good if you could remove this handlers each time the drawer is hidden.